src/pages/index.js

```diff
@@ -127,11 +203,16 @@ constants.addNewCardButton.addEventListener("click", () => {
   newCardPopup.open();
 });
 
-constants.cardDelete.forEach((button) => {
-  button.addEventListener("click", (event) => {
-    handleDeleteCard(event.target.closest(card));
-    console.log("Button clicked");
-  });
+constants.profileImageBtn.addEventListener("click", () => {
+  profilePicPopup.open();
+});
+
+constants.deleteImageClose.addEventListener("click", () => {
+  modalWithConfirm.close();
+});
+
+constants.deleteImageOverlay.addEventListener("click", () => {
+  modalWithConfirm.close();
```

Comment:
Please, delete it


Any close handling should be done by `Popup` because any modal is a child of `Popup`